datashader/transfer_functions.py

```diff
@@ -245,7 +247,7 @@ def set_background(img, color=None):
     if color is None:
         return img
     background = np.uint8(rgb(color) + (255,)).view('uint32')[0]
-    data = source(img.data, background)
+    data = over(img.data, background)
```

Comment:
On second look, I think I see what you're saying here, but I don't think it's true.  I.e., you're worried that the alpha channel will simply be discarded from the top image?  The code isn't discarding the src alpha, as far as I can see; it's using it to control how the src gets mixed with the background, which is what we want here.   Try comparing census.ipynb with set_background using over and source for comparison, and you should see what I mean...